src/main/java/frc/team2412/robot/subsystem/IntakeSubsystem.java

```diff
@@ -124,7 +125,7 @@ public void intakeStop() {
      */
     public void intakeExtend() {
         intakeSolenoidState = RETRACT;
-        solenoid.set(RETRACT.value);
+        // solenoid.set(RETRACT.value);
```

Comment:
Don't comment out - instead check if `solenoid` is non-null before using it, so you don't have to keep modifying this code.  Ditto for below.